app/controllers/chapters_controller.rb

```diff
@@ -172,6 +172,8 @@ def chapter_3
         { name: 'Terrestrial target (17%)', position: 23 }
       ]
     }.to_json
+
+    @line_chart_csv_url = URI.join(root_url, '/file/Ch3_Fig3_Global_PA_Timeseries.csv')
```

Comment:
Seeing as you've defined the constant in the initializer, this should be replaced with that